### ISSUE TITLE
All Campus Selection under Event Occurrence Makes Event Disappear for Campuses

### ISSUE THREAD

**Issue by jcamarin** [SparkDevNetwork/Rock#3355]
### Prerequisites
* [X] Put an X between the brackets on this line if you have done all of the following:
  * Can you reproduce the problem on a fresh install or the [demo site ](http://rock.rocksolidchurchdemo.com/)?
  * Did you include your Rock version number and [client culture ](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
  * Did you [%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack perform a cursory search ](https://github.com/issues?q=is) to see if your bug or enhancement is already reported?

### Description
When I choose 'All Campuses' under an event calendar occurrence instead of a specific campus, the event doesn't show when the calendar is filtered by campus (which happens by default on our site to whatever campus the person is assigned when logged in) The only way they can see all campus events is choosing 'no campus' under the campus filter, so we're having to enter multiple event occurrences for each of our campuses instead of one event for all campuses. 

1. Go to https://www.orchardcentral.info/events
1. Sort calendar filter dates for November 18, 2018. There is an event called Congregational Celebration that is assigned to All Campuses. 
1. Filter for the Cedar Falls Campus
1. See the event disappear.

**Expected behavior:**

When sorting for a specific campus, events that are assigned to 'All Campuses' should show up under every campus when filtered.

**Actual behavior:**

The event doesn't show up. So these All Campus events are some of our biggest events all year, but since the campus filter is set by default to the person's campus, they never show up on the calendar unless we make multiple event occurrences for every event per campus.

### Versions
* **Rock Version:** 8.3
* **Client Culture Setting:** en-us

**Comment by megilee** [comment]
I am also experiencing this issue with both the external public calendar and the internal staff calendar.

**Comment by jcamarin** [comment]
I am will pay someone to help get this fixed. It is so hard to put in all campus events because they are impossible to find on the event calendar as they don't show under each campus when the filter is being used. We have to do a lot of confusing work-arounds for these events. Can anyone help with this? I'd be happy to collaborate or write documentation or help however I can to get this bug fixed.

**Comment by KelleyLangkamp** [comment]
This appears to be fixed in v9.2. Filtering the calendar using both the filters in the left sidebar and using a campus url parameter both showed the event occurrences for all campuses.

**Comment by nairdo** [comment]
@jcamarin This was reported as resolved (in v9.2) in the #bugapalooza channel.

```
Kelley Langkamp (Rock Star) 3:54 PM
#3355 appears to be fixed
```
Please let us know by reopening if you feel this is closed in error.
